\chapter{SAM Format}
			\section{Introduction}
			SAM (Sequence Alignment/Map) format is a generic format for storing large nucleotide sequence alignments. SAM aims to be a format that: 
			\begin{itemize}
				\item Is flexible enough to store all the alignment information generated by various alignment programs;
				\item Is simple enough to be easily generated by alignment programs or converted from existing alignment formats;
				\item Is compact in file size;
				\item Allows most of operations on the alignment to work on a stream without loading the whole alignment into memory;
				\item Allows the file to be indexed by genomic position to efficiently retrieve all reads aligning to a locus.
			\end{itemize}
		
		SAM Tools provide various utilities for manipulating alignments in the SAM format, including sorting, merging, indexing and generating alignments in a per-position format.
		
		SAMtools is hosted by GitHub. The project page is here. The source code releases are available from the download page. You can check out the most recent source code with: 
		
		\begin{verbatim}
		git clone git://github.com/samtools/samtools.git
		\end{verbatim}